domains/tuannhat.json

```diff
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "nhathongphat",
+    "email": "nhatt@outlook.com"
+  },
+  "record": {
+    "CNAME": "nhathongphat.github.io"
```

Comment:
Little content provided on site. Reopen this PR when you have completed your site.